src/shared/components/unsupportedBrowserModal/UnsupportedBrowserModal.tsx

```diff
@@ -14,24 +14,21 @@ export default class UnsupportedBrowserModal extends React.Component<{}, IBrowse
     constructor() {
         super();
 
-        let state;
         if (!browser)  {
-            state = {
+            this.state = {
                 name: "Unsupported",
```

Comment:
@alisman yes. it works either way though because 2 lines before that, we set `name=name.toLowerCase()`. You can change it to lowercase in the state though for better clarity.